File: python/fizzbuzz.py
def fizzbuzz(n):
    if n != 0:
        if n % 3 == 0 and n % 5 == 0:
            return 'FizzBuzz'
        elif n % 5 == 0:
            return 'Buzz'
        elif n % 3 == 0:
            return 'Fizz'
    return n

File: ruby/fizzbuzz.rb
def fizzbuzz(number)
  return 'FizzBuzz' if number != 0 && (number % 3).zero? && (number % 5).zero?
  return 'Fizz' if  number != 0 && (number % 3).zero?
  return 'Buzz' if  number != 0 && (number % 5).zero?

  number
end
